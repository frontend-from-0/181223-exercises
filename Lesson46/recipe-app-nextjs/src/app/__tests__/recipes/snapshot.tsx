import { render } from "@testing-library/react";
import Page from "../../recipes/page";


it("renders homepage unchanged", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
