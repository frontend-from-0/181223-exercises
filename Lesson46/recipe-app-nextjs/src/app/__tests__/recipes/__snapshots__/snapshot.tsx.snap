// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h1
      data-testid="recipe-list-title"
    >
      Recipe list
    </h1>
  </div>
</div>
`;
